src/ParticleDA.jl

```diff
@@ -384,35 +399,9 @@ function update_particle_proposal!(model_data, filter_data, truth_observations,
 end
 
 function update_particle_proposal!(model_data, filter_data, truth_observations, nprt_per_rank, filter_type::OptimalFilter)
-
-        # Optimal Filter: After updating the particle dynamics, we apply the "optimal proposal" in
-        #                 sample_height_proposal!() to the first state variable (height). We apply
-        #                 a sample from the gaussian random field in update_particle_noise!() to the other
-        #                 state variables (velocity).
-
-        particles = get_particles(model_data)
-
-        # Apply optimal proposal, the result will be in offline_matrices.samples
-        sample_height_proposal!(@view(particles[:,:,1,:]),
```

Comment:
Quick note that `sample_height_proposal!` is unused, so it may be removed completely (also because as noted in https://github.com/Team-RADDISH/ParticleDA.jl/issues/208#issuecomment-1017245537 the name was too specific to the TDAC model)